include/nonstd/ring_span.hpp

```diff
@@ -480,6 +480,18 @@ class ring_span
 
     // element access:
 
+#if nsrs_RING_SPAN_LITE_EXTENSION
+    reference operator[]( size_type idx ) nsrs_noexcept
+    {
+        return assert( idx < m_size ), at_(idx);
+    }
+
+    const_reference operator[]( size_type idx ) const nsrs_noexcept
+    {
+        return assert( idx < m_size ), at_(idx);
```

Comment:
I believe that `assert` can't generally be assumed to work as an expression. Splitting it into its own statement would be cleaner. (And does this library even use `assert` for contract checking? I haven't looked.)

Reply:
https://github.com/martinmoene/ring-span-lite/blob/5ffa2f9368e5cf923713b143d229bc81c8fe3ab0/include/nonstd/ring_span.hpp#L446

Reply:
Thanks @Quuxplusone, changed in db1d64e9b665c2eb69ffb050d37a6468a91d511f

Reply:
@njlr, It’s just that it’s a habit to write in such a way that `assert` could work in constexpr with C++11.

For example https://github.com/martinmoene/string-view-lite/blob/88ad9b5b3771b2ceb01879ed3c8f296b3a0c9bdd/include/nonstd/string_view.hpp#L881

Reply:
remaining consistently inconsistent :)